src/css/main.less

```diff
@@ -1163,14 +1163,12 @@ dialog {
 }
 .tab_sponsor {
     display: none;
-    height: 50px;
-    max-height: 50px;
     margin: 0 auto 10px auto;
 }
 .img_sponsor {
     height: 100%;
-    padding-left: 10px;
-    padding-right: 10px;
+    max-height: 50px;
+    padding: 5px 10px;
```

Comment:
this change also adds additional 5px top and bottom padding. This to create a more spacing in between logos when they are displayed horizontally. I did not compensate this (by for instance removing padding 5px from padding of surrounding elements). But I can do that if requested (in order to keep negative space the same when displayed horizontally).